tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        VariationalInfer,
</pre><p>Organize imports and move to top of notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>